### PR TITLE
Add Kestrel aircraft icon

### PR DESCRIPTION
**Pull request by dspreitz**
This PR adds a didicated Kestrel map icon

### CODE REVIEW

**Comment by dspreitz** [comment]
Arg, this PR contains changes out of a different branch. I Need to fix this.

File: public/aircraft/kestrel.svg [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     inkscape:label="Ebene 1"
+     inkscape:groupmode="layer"
+     id="layer1"
+     transform="translate(0,-288.53334)">
+    <image
```
Comment (Turbo87) [inline public/aircraft/kestrel.svg]: this seems to be a PNG image embedded within an SVG file. can you make sure to make this an actual vector image? e.g. inkscape has a tool which can convert from pixels to vectors.

**Comment by Turbo87** [comment]
FYI the SVG file still included the PNG image. I removed it and ran the file through `svgo`, so it should be fine now.

**Comment by dspreitz** [comment]
Thanks for your final finishing touches on this and the merge.

**Pull request merged**